src/node_crypto.cc

```diff
@@ -741,6 +743,63 @@ void SecureContext::AddCRL(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void UseExtraCaCerts(const char* file) {
+  free(extra_root_certs_file);
+  extra_root_certs_file = file ? strdup(file) : nullptr;
+}
+
+static void EmitWarning(Environment* env, const char* fmt, ...) {
```

Comment:
This doesn't belong to `node_crypto.cc`. Perhaps `node.cc` and a declaration in `node_internals.h`?